keras/backend/tensorflow_backend.py

```diff
@@ -427,6 +428,7 @@ def constant(value, dtype=None, shape=None, name=None):
 
     # Returns
         A Constant Tensor.
+    {{np_implementation}}
```

Comment:
Same as the previous comment. Users will be confused since there is no `constant` in numpy.

Reply:
I got it. Thanks to advice.